python/py-gwdatafind/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-gwdatafind
+version             1.0.0
+
+categories-append   science
+maintainers         {ligo.org:duncan.macleod @duncanmmacleod} openmaintainer
+
+platforms           darwin
+license             GPL-3
+
+description         The client library for the LIGO Data Replicator (LDR) \
+                    service.
+long_description    The DataFind service allows users to query for the \
+                    location of Gravitational-Wave Frame (GWF) files \
+                    containing data from the current gravitational-wave \
+                    detectors.
+homepage            https://gwdatafind.readthedocs.io
+
+master_sites        pypi:g/gwdatafind
+distname            gwdatafind-${version}
+checksums   rmd160  d6ee9ee4ab579341543458acef32b6a12cf6d628 \
+            sha256  1441ad9dc445fc906bd3f589a059b878976958870cfad9ef20b9e2b8255378f4 \
+            size    31273
+
+python.versions     27 36
```

Comment:
in PR #2350 you suggest to add py37 to ```py-openssl```. After that py37 is supported for all dependencies here so you could add py37 as well. Just mention that the other PR should be merged first.